packages/backdrop/src/lib/backdrop.tsx

```diff
@@ -0,0 +1,27 @@
+import MUIBackdrop, {
+  BackdropProps as MUIBackdropProps,
+} from '@mui/material/Backdrop';
+import { ReactNode } from 'react';
+
+/* eslint-disable-next-line */
+export interface BackdropProps extends MUIBackdropProps {
+  children: ReactNode;
+}
+
+export function Backdrop(props: BackdropProps) {
+  const { children, ...other } = props;
+
+  return (
+    <MUIBackdrop
+      {...other}
+      sx={{
+        zIndex: (theme) => theme.zIndex.drawer + 1,
+        backgroundColor: 'rgba(128, 128, 128, 0.5) ',
```

Comment:
we have a function addTransparency at @novatics/styles

Reply:
Fixed on new commit